app/_component/FavoriteBoardSection.tsx

```
@@ -69,33 +70,39 @@ export default function FavoriteBoardSection({
-                {Object.keys(posts).length === 0 
-                  ? "로그인이 필요합니다" 
-                  : `"${boards.find(b => b.key === selected)?.label}" 게시글이 없습니다`
-                }
-              </p>
-            </div>
-          )}
+          {useMemo(() => {
+            const list = posts[selected] ?? [];
+            if (list.length === 0) {
+              return (
+                <div className="bg-white border border-gray-200 rounded-xl p-4 text-center">
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)